src/containers/admin-dashBoard/ManagerRoles.tsx

```diff
@@ -269,69 +278,130 @@ const AdminSission = () => {
                       </tr>
                     </thead>
                     <tbody>
-                      {developers.map((developer: any, index: any) => {
-                        let rowTheme =
-                          index % 2 !== 0
-                            ? 'bg-light-bg dark:bg-dark-tertiary'
-                            : 'bg-white dark:bg-dark-bg';
+                      {data
+                        .slice(firstContentIndex, lastContentIndex)
+                        .map((item: any, index: any) => {
+                          let rowTheme =
+                            index % 2 !== 0
+                              ? 'bg-light-bg dark:bg-dark-tertiary'
+                              : 'bg-white dark:bg-dark-bg';
 
-                        return (
-                          <tr className={`${rowTheme}`} key={index}>
-                            <td className="border-b border-gray-200 py-2">
-                              <div className="flex items-left">
-                                <span className="hidden  ml-2 md:inline-block h-10 w-10 rounded-full overflow-hidden bg-gray-100 dark:bg-dark-tertiary">
-                                  <svg
-                                    className="h-full w-full text-gray-300 dark:text-white"
-                                    fill="currentColor"
-                                    viewBox="0 0 24 24"
-                                  >
-                                    <path d="M24 20.993V24H0v-2.996A14.977 14.977 0 0112.004 15c4.904 0 9.26 2.354 11.996 5.993zM16.002 8.999a4 4 0 11-8 0 4 4 0 018 0z" />
-                                  </svg>
-                                </span>
-                                <div className="flex flex-col  leading-4 px-3 py-2">
-                                  <h3 className="dark:text-white sm:text-xs">
-                                    {developer.name}
-                                  </h3>
-                                  <p className="text-sm sm:text-xs text-gray-400 dark:text-white">
-                                    {developer.email}
-                                  </p>
+                          let Data;
+                          if (
```

Comment:
Please re-check the way you rebased. You have affected previous features of @chanceDira